podpac/core/cache/test/test_cache_stores.py

```diff
@@ -358,7 +358,7 @@ def test_size(self):
         assert store.size == expected_size
 
 
-@pytest.mark.aws
+@pytest.mark.skipif(pytest.config.getoption("--ci"), reason="skip AWS tests during CI")
```

Comment:
@jmilloy When I was doing the black merge, I wasn't sure which form was "correct". Please fix this if I got it wrong.

Reply:
Should be the `@pytest.mark.aws` version going forwar. The `pytest.config` global is deprecated. I'll fix that.